src/common/hook/usePublishQueue/utils/publish.tsx

```diff
@@ -94,22 +96,30 @@ export const groupDocumentsIntoJobs = (
   const transferableRecords = rawDocuments.filter((doc) => doc.type === "TRANSFERABLE_RECORD");
   const verifiableDocuments = rawDocuments.filter((doc) => doc.type === "VERIFIABLE_DOCUMENT");
   const groupedVerifiableDocuments = groupBy(verifiableDocuments, "contractAddress");
-  const documentStoreAddresses = Object.keys(groupedVerifiableDocuments);
+  const verifiableDocumentsWithDocumentStore = { ...groupedVerifiableDocuments };
+  delete verifiableDocumentsWithDocumentStore["DNS-DID"];
+  const verifiableDocumentsWithDnsDid =
+    Object.keys(groupedVerifiableDocuments).indexOf("DNS-DID") >= 0
+      ? [...groupedVerifiableDocuments["DNS-DID"]]
+      : [];
+  const documentStoreAddresses = Object.keys(verifiableDocumentsWithDocumentStore);
```

Comment:
||

Reply:
the || dont seem to work, it will still throw an error cos it tried to iterate the array. and fail to iterate the array.